### PR TITLE
Notification high

### PR DESCRIPTION
**Pull request by koverchik**

### CODE REVIEW

**Review by DimaKelek — APPROVED** [review]

File: agoraStreaming/src/Components/ModalCreateEvent/style.ts [hunk 1]
```diff
@@ -11,12 +11,12 @@ type LifeScreenStyles = {
   modalView: ViewStyle;
   input: ViewStyle;
   errorInput: ViewStyle;
-  addNewEvent: ViewStyle;
+
```
Comment (Mentos15) [inline agoraStreaming/src/Components/ModalCreateEvent/style.ts:14]: delete empty line

File: agoraStreaming/src/Screens/Calendar/index.tsx [hunk 2]
```diff
@@ -16,18 +16,20 @@ import {
...
   const [chosenDay, setChoseDay] = useState(
     `${dataSystem.getFullYear()}-${
       dataSystem.getMonth() + 1
     }-${dataSystem.getDate()}`,
   );
 
   const [streams, setStreams] = useState<StreamType[]>([]);
-
+  const changeModalVisible = () => setModalVisible(!isModalVisible);
   useEffect(() => {
```
Comment (Mentos15) [inline agoraStreaming/src/Screens/Calendar/index.tsx:33]: add empty line

File: agoraStreaming/src/Screens/Calendar/index.tsx [hunk 3]
```diff
@@ -66,11 +67,16 @@ export const ScreenCalendar: FC = () => {
...
   return (
     <View style={styles.background}>
       <View style={styles.container}>
+        <TouchableOpacity
+          onPress={changeModalVisible}
+          style={styles.addNewEvent}>
+          <FontAwesomeIcon icon={faPlus} color={'white'} size={18} />
+        </TouchableOpacity>
         <Calendar
           onDayPress={(day) => {
```
Comment (Mentos15) [inline agoraStreaming/src/Screens/Calendar/index.tsx:81]: in my opinion better create a function and use function

File: agoraStreaming/src/Screens/Calendar/styles.ts [hunk 4]
```diff
@@ -1,10 +1,13 @@
 import {StyleSheet, ViewStyle} from 'react-native';
 
+import {SIZE_CLOSE_BUTTON as SIZE_BUTTON} from '../../Components/ModalCreateEvent/style';
```
Comment (Mentos15) [inline agoraStreaming/src/Screens/Calendar/styles.ts:3]: delete "as SIZE_BUTTON"

**Pull request merged**